README.md

```diff
@@ -16,201 +16,18 @@ This repository contains both the core rendering library and sample applications
-* For Device: You will have to modify the _Bundle Identifier_ under target _tangram_ > _General_ > _Identity_ > _Bundle Identifier_, to something other than `com.mapzen.tangram`, since this needs to be unique.
-
-
-### iOS Binary Framework ###
-
-An iOS binary framework bundle targeted for ARM architectures can be produced by running the following:
-
-```bash
-make ios-framework [RELEASE=1|DEBUG=1]
-```
-
-The framework will be available in the configuration build type in `/build/ios-framework/lib/`.
-
-To build a universal binary working on both device and simulator architectures run the following:
-
-```bash
-make ios-framework-universal [RELEASE=1|DEBUG=1]
-```
-
-The universal framework will be available in the configuration build type in `/build/ios-framework-universal/`.
-
-### Android ###
-To build for Android you'll need to use [Android Studio](https://developer.android.com/studio/index.html) version **2.2** or newer on Mac OS X, Ubuntu, or Windows 10. Using the Android Studio SDK Manager, install or update the 'CMake', 'LLDB', and 'NDK' packages from the 'SDK Tools' tab. Once dependencies are installed, you can execute Android builds from either the command line or the Android Studio interface.
-
-To build the demo application for the ARMv7 architecture (covers most Android devices), run:
-
-```bash
-make android
-```
-
-Or open the project in Android Studio and press the 'Run' button (^R). More options are provided through Gradle.
-
-The Gradle project in the `android/` directory contains two modules: a library module called `tangram` containing the Tangram Android SDK and an application module called `demo`, containing a demo application that uses the `tangram` module. The `tangram` module has two `buildTypes`, `debug` and `release`, and two `productFlavors`, `slim` and `full`. The `slim` flavor includes native libraries for just the ARMv7 architecture, the `full` flavor includes all supported architectures (ARMv6, ARMv7, ARM64, and x86).
-
-To build the library or demo application from the `android/` folder using Gradle, use the conventional syntax, e.g.:
-
-```bash
-./gradlew tangram:assembleFullRelease
-```
-
-Android Studio supports debugging both the Java and C++ parts of tangram-es on a connected device or emulator. Choose one of the 'debug' build variants, set your desired breakpoints, and press the 'Debug' button (^D).
+Currently we are targeting five platforms:
 
-### Raspberry Pi ###
-To build on Rasberry Pi you will need a C++ toolchain with support for C++14. GCC 4.9 (or higher) is known to work (refer [here](https://community.thinger.io/t/starting-with-the-raspberry-pi/36) for instructions on getting GCC 4.9).
+ - [Android](platforms/android)
+ - [iOS](platforms/ios)
+ - [Mac OS X](platforms/osx)
+ - [Ubuntu Linux](platforms/linux)
+ - [Raspberry Pi](platforms/rpi)
 
-First, install CMake and libcurl:
+## Code Style ##
```

Comment:
We could add that in a `Contributing.md` as mentioned in https://github.com/tangrams/tangram-es/issues/692.